ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/autovpn-edge-no-default-policy.cfg

```diff
@@ -101,6 +102,13 @@ application traffic recognition
    field-set ipv4 prefix CONTROL-PLANE-APP-DEST-PREFIXES
       2.2.2.2/32
 !
+ip access-list ACL-WAN-TRANSPORT-OUT
+   10 remark  ### Blocking RFC 1918 prefixes ###
+   20 deny ip any 10.0.0.0/8
+   30 deny ip any 172.16.0.0/12
+   40 deny ip any 192.168.0.0/16
```

Comment:
This would only apply on internet circuits. We may have to at least permit the interface address itself at the top. @gmuloc.